planner/core/task.go

```diff
@@ -1021,6 +1021,17 @@ func (p *PhysicalProjection) GetCost(count float64) float64 {
 }
 
 func (p *PhysicalProjection) attach2Task(tasks ...task) task {
+	if cop, ok := tasks[0].(*copTask); ok {
```

Comment:
we should check if every expression can be pushed.

Reply:
updated